app/config.py

```diff
@@ -366,9 +366,10 @@ class Config(object):
                 "schedule": crontab(hour=9, minute=0, day_of_week="tue", day_of_month="1-7"),
                 "options": {"queue": QueueNames.REPORTING},
             },
+            # first wednesday of every month
             "change-dvla-password": {
                 "task": "change-dvla-password",
-                "schedule": crontab(hour=9, minute=5, day_of_week="tue", day_of_month="1-7"),
+                "schedule": crontab(hour=9, minute=0, day_of_week="wed", day_of_month="1-7"),
```

Comment:
This could in theory run on the 1st of the month and then the api key changes on the 7th 🤔 

That is of course fine, but if we wanted to make sure they're consecutive I think we could change `day_of_month="2-8"`?

Reply:
good point. thats probably less surprising

Reply:
🦶 squashed out